app/classifier/drawing-tools/point.cjsx

```diff
@@ -40,6 +43,10 @@ module.exports = React.createClass
     x: (SELECTED_RADIUS / @props.scale.horizontal) * Math.cos theta
     y: -1 * (SELECTED_RADIUS / @props.scale.vertical) * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-point'
+    deleteIfOutOfBounds target
+
```

Comment:
This throws `Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined`. Looks like the React component is being passed in as `target`, but `deleteIfOutOfBounds` expects a DOM element.